crates/bevy_render/src/view/mod.rs

```diff
@@ -218,12 +218,27 @@ impl ViewTarget {
         &self.out_texture
     }
 
+    /// the initial texture that will be written to.
+    pub fn base_texture(&self) -> &TextureView {
+        self.sampled_main_texture().unwrap_or(&self.main_textures.a)
+    }
+
+    /// is the last written texture also the base texture
+    pub fn is_current_base(&self) -> bool {
+        self.sampled_main_texture().is_none() && self.main_texture.load(Ordering::SeqCst) == 0
+    }
+
     /// The format of the final texture this view will render to
     #[inline]
     pub fn out_texture_format(&self) -> TextureFormat {
         self.out_texture_format
     }
 
+    /// the format of the initial texture that will be written to.
+    pub fn base_texture_format(&self) -> TextureFormat {
```

Comment:
Why not just use `main_texture_format()`? Do you think these might be different in the future?